dbms/src/Storages/Page/V3/WALStore.cpp

```diff
@@ -221,8 +220,24 @@ bool WALStore::saveSnapshot(FilesSnapshot && files_snap, PageEntriesEdit && dire
 #endif
         }
     }
-    // TODO: Log more information. duration, num entries, size of compact log file...
-    LOG_FMT_INFO(logger, "Save directory snapshot to log file done [num_compacts={}]", files_snap.persisted_log_files.size());
+
+    FmtBuffer fmt_buf;
+    fmt_buf.append("Dumped snapshotSave directory snapshot to log file done. [files_snapshot: ");
```

Comment:
```suggestion
    fmt_buf.append("Dumped directory snapshot to log file done. [files_snapshot=");
```

Reply:
will change in #4534 

current pr base on it...